components/learn/modules/ROOT/pages/deploy-and-interact.adoc

```diff
@@ -147,13 +147,13 @@ NOTE: Remember that local blockchains **do not** persist their state throughout
 [[interacting-via-the-cli]]
 == Interacting via the CLI
 
-With our `Box` contract <<deploying-a-smart-contract, deployed>>, we can start using it right away from the <<getting-started-with-the-cli, CLI>>.
+With our `Box` contract <<deploying-a-smart-contract, deployed>>, we can start using it right away from the CLI.
```

Comment:
Why remove the link?

Reply:
It felt superfluous when I reviewed, but as the CLI is two sections previously, fine to leave as is.